installPhase.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+if [ -v out ]; then
+  echo "Will install package to $out (defined outside installPhase.sh script)"
+else
+  out="./result"
+  echo "Will install package to $out"
+fi
+
+COMMIT_SHORT=$(git rev-parse --short HEAD)
+VERSION_APPENDIX=""
+if [ -n "$COMMIT_SHORT" ]; then
+    VERSION_APPENDIX="-$COMMIT_SHORT"
+else
+    VERSION_APPENDIX="-NOGIT"
+fi
+
+# Extract version from Cargo.toml using toml2json
+PACKAGE_VERSION=$(toml2json < Cargo.toml | jq -r .package.version)
+if [ -z "$PACKAGE_VERSION" ]; then
+    echo "Could not extract version from Cargo.toml"
+    exit 1
+fi
+PACKAGE_VERSION+=$VERSION_APPENDIX
```

Comment:
This is unnecessary as we enforce that `Cargo.toml` and `package.json` are kept in sync with release-please. We could rework this into a CI action to check this condition however.